spec/acceptance/nodesets/default.yml

```diff
@@ -1,3 +1,10 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
```

Comment:
Need to change this to `SIMP_BEAKER_HYPERVISOR`

Reply:
`BEAKER_HYPERVISOR` is already present in quite a few modules, I wouldn't like to change it now.